src/lib/controllers/ui-controller.js

```diff
@@ -1240,7 +1240,7 @@ export default class UIController {
     if (this.api.ui.hasModule('actionPanel')) {
       this.store.state.actionPanel.visible
         ? this.store.commit('actionPanel/close')
-        : this.store.commit('actionPanel/open')
+        : this.store.commit('actionPanel/open', {})
```

Comment:
I was to add this fix, otherwise compact toolbar was broken (it is because of some last PR - it is a new bug - and it is in msater now)